docs/user_guides/fs/storage_connector/creation/snowflake.md

```diff
@@ -16,18 +16,39 @@ When you're finished, you'll be able to query the database using Spark through H
 
 Before you begin this guide you'll need to retrieve the following information from your Snowflake account and database, the following options are **mandatory**:
 
-- **Snowflake Connection URL:** Consult the documentation of your target snowflake account to determine the correct connection URL. This is usually some form of your [Snowflake account identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html). For example:
+- **Snowflake Connection URL:** Consult the documentation of your target snowflake account to determine the correct 
+  connection URL. There are multiple formats supported by Snowflake. This is usually some form of your [Snowflake account 
+  identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html) or an 
+[account URL](https://docs.snowflake.com/en/user-guide/organizations-connect.html#connecting-with-a-url).
 
+For example, connection URL can be set using an account 
+  identifier format:
 ```
 <account_identifier>.snowflakecomputing.com
 ```
+OR using any of the account URL format:
+```
+https://<orgname>-<account_name>.snowflakecomputing.com
```

Comment:
Have you test this format? I'm highly skeptical this works. Considering the format above, I think somewhere in the codebase we prepend `https://` to the string. 
I suspect if you put the above, you'll end up with `https://https://<orgname>-<account_name>...`

Reply:
I tested this on app, I used the account URL copied from snowflake and it worked in the import data functionality.
In the code, we do not seem to prepend anything in the storage connector class, the url is passed to spark options as is.

Reply:
keeping this example format, please let me know if this also should be removed.